tests/integration/runme.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+export ANSIBLE_STDOUT_CALLBACK=foreman
+export ANSIBLE_LOAD_CALLBACK_PLUGINS=1
+export ANSIBLE_FORCE_COLOR=0
+export ANSIBLE_NOCOLOR=1
+# No connection is actually performed during the test
+export FOREMAN_URL=http://localhost
+export FOREMAN_SSL_CERT=/dev/zero
+export FOREMAN_SSL_KEY=/dev/zero
+export FOREMAN_DIR_STORE=/tmp
+
+ansible-playbook -i inventory test.yml
+
+sed -E -i 's/"[0-9]+-[0-9]+-[0-9]+[ T][0-9]+:[0-9]+:[0-9]+\.[0-9]+/"2000-01-01 12:00:00.0000/g' /tmp/*.json
+sed -E -i 's/"[0-9]+:[0-9]+:[0-9]+\.[0-9]+/"0:00:00.01/g' /tmp/*.json
```

Comment:
These statements are to clear date and time strings so fixtures can be compared. Probably an ENV variable could be used to generate new fixtures.

There is no test code yet, TBD.